src/Uno.UI/UI/Xaml/Media/Imaging/RenderTargetBitmap.skia.cs

```diff
@@ -0,0 +1,59 @@
+﻿#nullable enable
+using System.Runtime.InteropServices;
+using Windows.Foundation;
+namespace Windows.UI.Xaml.Media.Imaging
+{
+	partial class RenderTargetBitmap
+	{
+		delegate void SwapColor(ref byte[] buffer, int byteCount);
+
+
+		static readonly SwapColor PlatformSwap = RuntimeInformation.IsOSPlatform(OSPlatform.OSX)
+			? new SwapColor(SwapRB)
+			: new SwapColor(DoNothing);
```

Comment:
For performance purposes, you should make the delegate `null` instead of calling one that does nothing.